web/themes/custom/cloudy/src/components/morsel/morsel.twig

```diff
@@ -24,7 +24,7 @@
         classes: ['mb-2'],
       } only %}
     {% endif %}
-    {% if updated_on and (updated_on|render|striptags|date('U') > posted_on|render|striptags|date('U')) %}
+    {% if updated_on and (updated_on|striptags|date('U') > posted_on|striptags|date('U')) %}
```

Comment:
@rockymountainhigh1943, I noticed that news on https://powr-2123-2-portlandor.pantheonsite.io/news is getting the morsel-updated class even if the updated and posted dates are identical. Is that intentional? Do we need .morsel--updated if the styling associated with an updated date is at the field level?